repository/src/main/java/org/apache/atlas/repository/store/aliasstore/ESAliasStore.java

```diff
@@ -192,7 +192,12 @@ private void personaPolicyToESDslClauses(List<AtlasEntity> policies,
                     }
 
                     for (String asset : assets) {
-                        terms.add(asset);
+                        if (asset.contains("*") || asset.contains("?")) {
```

Comment:
Let's also check `policyResourceCategory: "CUSTOM"` in this if condition

Reply:
Any reason? asking since we validate policyResourceCategory while creating Persona metadata, glossary policies, I think we do not need this check here